coralproject/talk
# [CORL-2891]: fix edit child reply to edited parent not updating

## What does this PR do?

These changes make it so that we only check that the latest parent comment revision matches up with the parent revision id input when creating a comment reply. Previously, we were also checking whenever retrieving a parent comment (including when a comment is edited). 

This makes it so that child replies, to a parent comment that is edited after the reply has been created, can still be edited themselves.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags?

none

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

Create a comment. Create a reply to the comment. Edit the parent comment. Edit the reply comment. See that everything is edited/updated as expected and you get no errors.

## Where any tests migrated to React Testing Library?



## How do we deploy this PR?
